doc/introduction/remote.rst

```diff
@@ -0,0 +1,511 @@
+S2gate(0, 0) | (q[1], q[5])
+MZgate(0, 5.236) | (q[0], q[1])
+MZgate(4.886, 5.496) | (q[1], q[2])
+MZgate(0.7106, 4.492) | (q[2], q[3])
+Rgate(0.9284) | (q[3])
+MZgate(2.922, 3.142) | (q[0], q[1])
+MZgate(4.528, 3.734) | (q[1], q[2])
+Rgate(-2.51) | (q[2])
+MZgate(0, 5.236) | (q[4], q[5])
+MZgate(4.886, 5.496) | (q[5], q[6])
+MZgate(0.7106, 4.492) | (q[6], q[7])
+Rgate(0.9284) | (q[7])
+MZgate(2.922, 3.142) | (q[4], q[5])
+MZgate(4.528, 3.734) | (q[5], q[6])
+Rgate(-2.51) | (q[6])
+Rgate(-2.51) | (q[1])
+Rgate(-0.8273) | (q[4])
+Rgate(-0.8273) | (q[0])
+Rgate(-2.51) | (q[5])
+MeasureFock | (q[0], q[1], q[2], q[3], q[4], q[5], q[6], q[7])
+
+The squeezing values required to embed this bipartite graph are given by
+the following relation:
+
+>>> from thewalrus.quantum import find_scaling_adjacency_matrix
+>>> c = find_scaling_adjacency_matrix(A, 2*4*0.345274461385554870545)
+>>> set(np.arctanh(np.linalg.svd(c*A)[1]))
+{0.0, 1.0000000000000002}
+
+Note that the above squeezing values must be of the form :math:`\{0,1\}`
+to be embedded on the chip. Consider a bipartite graph where this is not
+the case:
+
+>>> B = np.array([[0, 1, 0, 1], [1, 0, 1, 0], [0, 1, 1, 1], [1, 0, 1, 0]])
+>>> A = np.block([[0*B, B], [B.T, 0*B]])
+>>> c = find_scaling_adjacency_matrix(A, 2*4*1)
+>>> set(np.arctanh(np.linalg.svd(c*A)[1]))
+{0.0,
+3.2937343775007984e-32,
+0.17674864137317442,
+0.17674864137317453,
+0.8180954232791708,
+0.8180954232791715,
+1.3361892276414615}
+
+The program will fail to compile for chip2:
+
+.. code-block:: python3
+
+    prog = sf.Program(8)
+
+    with prog.context as q:
+        ops.BipartiteGraphEmbed(A, mean_photon_per_mode=1) | q
+        ops.MeasureFock() | q
+
+    prog.compile("chip2").print()
+
+.. code-block:: bash
+
+    ---------------------------------------------------------------------------
+
+    CircuitError                              Traceback (most recent call last)
+
+    <ipython-input-23-f713320d8c3b> in <module>
+          5     ops.MeasureFock() | q
+          6
+    ----> 7 prog.compile("chip2").print()
+
+
+    ~/Dropbox/Work/Xanadu/sf_cloud/strawberryfields/program.py in compile(self, target, **kwargs)
+        522         # does the circuit spec  have its own compilation method?
+        523         if db.compile is not None:
+    --> 524             seq = db.compile(seq, self.register)
+        525
+        526         # create the compiled Program
+
+
+    ~/Dropbox/Work/Xanadu/sf_cloud/strawberryfields/circuitspecs/chip2.py in compile(self, seq, registers)
+        137             raise CircuitError(
+        138                 "Incorrect squeezing value(s) (r, phi)={}. Allowed squeezing "
+    --> 139                 "value(s) are (r, phi)={}.".format(wrong_params, allowed_sq_value)
+        140             )
+        141
+
+
+    CircuitError: Incorrect squeezing value(s) (r, phi)={(1.336, 0.0), (0.177, 0.0), (0.818, 0.0)}. Allowed squeezing value(s) are (r, phi)={(1, 0.0), (0.0, 0.0)}.
```

Comment:
Enough to just reproduce the error message, not the full traceback (which anyway contains some unnecessary developer-specific stuff)